{ubuntu-questing,debian-sid}-standard: maxima FTBFS

```
  [maxima-5.47.0]   [spkg-install] ;;; Compiling /sage/local/var/tmp/sage/build/maxima-5.47.0/src/src/maxima-package.lisp.
  [maxima-5.47.0]   [spkg-install] ;;; OPTIMIZE levels: Safety=2, Space=0, Speed=3, Debug=2
  [maxima-5.47.0]   [spkg-install] ;;;
  [maxima-5.47.0]   [spkg-install] ;;; Internal error:
  [maxima-5.47.0]   [spkg-install] ;;;   ** Error code 1 when executing
  [maxima-5.47.0]   [spkg-install] ;;; (EXT:RUN-PROGRAM "gcc" ("-I." "-I/usr/include/" "-Wdate-time" "-D_FORTIFY_SOURCE=3" "-D_GNU_SOURCE" "-D_FILE_OFFSET_BITS=64" "-g" "-O3" "-Werror=implicit-function-declaration" "-fno-omit-frame-pointer" "-mno-omit-leaf-frame-pointer" "-ffile-prefix-map=/build/ecl-kdeNWB/ecl-24.5.10+ds=." "-fstack-protector-strong" "-fstack-clash-protection" "-Wformat" "-Werror=format-security" "-fcf-protection" "-fdebug-prefix-map=/build/ecl-kdeNWB/ecl-24.5.10+ds=/usr/src/ecl-24.5.10+ds-1" "-fPIC" "-D_THREAD_SAFE" "-Dlinux" "-O2" "-c" "binary-ecl/maxima-package.c" "-o" "binary-ecl/maxima-package.o")):
  [maxima-5.47.0]   [spkg-install] ;;; In file included from /usr/include/ecl/ecl.h:81,
  [maxima-5.47.0]   [spkg-install] ;;;                  from /usr/include/ecl/ecl-cmp.h:31,
  [maxima-5.47.0]   [spkg-install] ;;;                  from binary-ecl/maxima-package.c:5:
  [maxima-5.47.0]   [spkg-install] ;;; /usr/include/ecl/object.h:27:13: error: 'bool' cannot be defined via 'typedef'
  [maxima-5.47.0]   [spkg-install] ;;;    27 | typedef int bool;
  [maxima-5.47.0]   [spkg-install] ;;;       |             ^~~~
  [maxima-5.47.0]   [spkg-install] ;;; /usr/include/ecl/object.h:27:13: note: 'bool' is a keyword with '-std=c23' onwards
  [maxima-5.47.0]   [spkg-install] ;      - Binary file binary-ecl/maxima-package.fas is old or does not exist. 
  [maxima-5.47.0]   [spkg-install] ;        Compile (and load) source file /sage/local/var/tmp/sage/build/maxima-5.47.0/src/src/maxima-package.lisp instead? y
  [maxima-5.47.0]   [spkg-install] ;      - Should I bother you if this happens again? y
```
https://github.com/passagemath/passagemath/actions/runs/16915470407/job/47928135110#step:11:10232

--- a/build/pkgs/ecl/spkg-configure.m4
+++ b/build/pkgs/ecl/spkg-configure.m4
@@ -4,8 +4,9 @@ SAGE_SPKG_CONFIGURE([ecl], [
   sage_spkg_install_ecl=yes
 
   SAGE_SPKG_DEPCHECK([gcc gc gmp], [
+    AC_PATH_PROG([ECL], [ecl])
     AC_PATH_PROG([ECL_CONFIG], [ecl-config])
-    AS_IF([test x$ECL_CONFIG != x], [
+    AS_IF([test x$ECL != x -a x$ECL_CONFIG != x], [dnl
       # "CPPFLAGS" is not a typo, the --cflags output from
       # ecl-config typically contains -D and -I flags.
       saved_CPPFLAGS="${CPPFLAGS}"
@@ -16,11 +17,16 @@ SAGE_SPKG_CONFIGURE([ecl], [
         #include <ecl/config.h>
       ]],[[
         if (ECL_VERSION_NUMBER < 210201) { return 1; }
-      ]])],
-      [
-        sage_spkg_install_ecl=no
-      ],
-      [
+      ]])], [dnl
+        rm -rf conftest.*
+        cat > conftest.lisp <<EOF
+(defun foo ()
+  nil)
+EOF
+        AS_IF([$ECL --norc --eval '(multiple-value-bind (output-truename warnings-p failure-p) (compile-file "conftest.lisp" :system-p t) (if failure-p (quit 1) (quit)))' 2>& ]AS_MESSAGE_LOG_FD[ >&2], [dnl
+          sage_spkg_install_ecl=no
+        ])
+      ], [dnl
         CPPFLAGS="${saved_CPPFLAGS}"
         AC_MSG_NOTICE([ecl found but too old])
       ])

